### DISCUSSION TITLE
patchs

### DISCUSSION THREAD

**Discussion by gian956** [Mr-MIBonk/M.I.B._More-Incredible-Bash#154] [General]
buongiorno, non riesco a trovare la patch per la versione MHI2_ER_AUG22_S2133 MU0358 grazie per l'aiuto

**Comment by harman-f** [comment]
Update FW and patch.

**Reply by gian956** [reply]
purtroppo non riesco a scaricare il FW, non mi completa il download

**Reply by tommybol** [reply]
Use a download manager, this will speed up your download ( free download manager worked for me )

**Reply by gian956** [reply]
grazie !!!!!!! download completato

**Reply by gian956** [reply]
aggiornamento fatto, ma ora ho perso l'audio bose, aiuto!!!
